src/types/IControlInfo.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <wtypes.h>
+
+namespace Microsoft::Console::Types
+{
+    class IControlInfo

```

Comment:
This may need a slightly clearer name; `IControlAccessibilityInfo`? `ITerminalAccessibilityInfo`?

Reply:
How's this differ from IUiaData(?)

Reply:
This interface was for information specific to the hosting control. IUiaData is implemented on the internal terminal class. Which doesn't have data on things like padding or the host uia provider.

Reply:
Huge thanks. This is a nice way to know what _exactly_ needs to be implemented in a new control.